module_utils/network_lsr/nm/client.py

```diff
@@ -83,4 +87,5 @@ def fail(self, exception):
 
 
 def get_mainloop(timeout):
-    return _NmMainLoop(timeout)
+    if HAS_NM:
+        return _NmMainLoop(timeout)
```

Comment:
Same with this - we don't need to check `HAS_NM`

Reply:
@nhosoi is there a risk that the function will be actually executed on the control node? If not, then indeed we don't need to check `HAS_NM`, right?

Reply:
@richm, @pcahyna, ok. Let me remove the `if HAS_NM:` check and see how it goes...

Reply:
Yep, you are right, @richm, @pcahyna. We don't need the check. Let me squash the test commit into the main one.
```
Testing /tmp/tmpvdv8w27n/tests/network/tests_802_1x_nm.yml...SUCCESS
```

Reply:
Thanks, @richm, @pcahyna. You are right. By just ignoring the ValueError cause by the missing NetworkManager package, the CI tests in the collections format passes. This is the commit message explaining it.
```
2) In the later module_utils path finding path, gi.require_version("NM", "1.0")
   in module_utils/network_lsr/nm/client.py fails with "ValueError: Namespace
   NM not available" on the control node. By ignoring the exception, the failure
   is worked around. Please note that the missing package issue never occurs
   on the managed nodes since in case of "nm", the NetworkManager package is
   installed in the network role.
```